src/components/Events/Events.jsx

```diff
@@ -39,20 +51,20 @@ const Events = () => {
             <MainEventsContainer>
                 <EventsHeading>Today's Events</EventsHeading>
```

Comment:
The issue number #190 is not getting resolved with.

![image](https://user-images.githubusercontent.com/44284877/196005809-aa182d91-deb7-4941-a8f5-bd4f79a8480d.png)

because today's events are outside of the map / loop functions

so where there are no event's on today, so we want to display that heading too